src/backend/dx11/Cargo.toml

```diff
@@ -30,9 +30,9 @@ name = "gfx_device_dx11"
 [dependencies]
 log = "0.3"
 gfx_core = { path = "../../core", version = "0.7" }
-d3d11-sys = "0.2"
-d3dcompiler-sys = "0.2"
-dxgi-sys = "0.2"
-dxguid-sys = "0.2"
-winapi = "0.2"
+d3d11-sys = { git = "https://github.com/msiglreith/winapi-rs.git", branch = "gfx" }
```

Comment:
Yes or a new 0.2 release, only a small patch is required. I don't know what @retep998 would prefer.